nixos/maintainers/scripts/oci/upload-image.sh

```diff
@@ -0,0 +1,98 @@
+#! /usr/bin/env bash
```

Comment:
For what it is worth, after working with the AWS uploader for some time, I have needed to reimplement it with Terraform to keep track of the details. I wonder if using Terraform for this instead of a script would work too?

Reply:
I think Terraform would work for this too. The oci terraform provider seems to have the functionality needed to import a custom image: https://registry.terraform.io/providers/hashicorp/oci/latest/docs/resources/core_image

Would you mind sharing what details needed to be tracked in your use-case for AWS? I can't currently write/test Terraform code as my free trial has expired which unfortunately prevents me from importing custom images.